en/content_how_to_get_compensated_for_work.md

```diff
@@ -1,47 +1,71 @@
 # How to Get Compensated for Work
 
-## Where do I Find Work?
+The [Smart Contract Research Forum](http://smartcontractresearch.org) (SCRF) created How to Get Compensated for Work to guide contributors through SCRF's payment system.
 
-Look for announcements on the [Smart Contract Research Forum](https://www.smartcontractresearch.org/), check [Github](https://github.com/orgs/smartcontractresearchforum/projects/5) for active projects that interest you, or ask directly, by going into the [Chat](https://discord.gg/nJDwxv72) and asking how to get involved.
+## Overview
 
-## What Kind of Work is Available at SCRF?
+This document covers how to create an invoice, track hours, alternative payment methods, and important notes. Recently onboarded contributors and recipients of grants, awards, or rewards need to follow the steps below to receive compensation.
 
-There’s a wide variety of work at SCRF, from writing Research Summaries to copy editing and graphic design to various administrative roles. We support flexible disbursement schedules, honoraria, grants, and other arrangements.
+## Invoicing and Payment System
 
-As researchers become more deeply engaged with projects, or take on larger pieces of work, we have longer-term relationships available, including salaried positions.
+SCRF officially pays invoices at the end of each month, after the work has been performed and billed. In certain situations, SCRF may process another round of payments mid-month. **All invoices are due five full business days before the end of the month**. Any billable hours for the month that were forgotten or performed after the invoice was submitted can be added to the next invoice with a description. SCRF uses Request Network for all payments.
 
-_We pay in cryptocurrency. For longer commitments, alternative arrangements can be made._
+How to get set up:
 
-If you cannot accept funds due to organizational or legal reasons, we can discuss alternative options. This could include:
+* Use an Ethereum wallet address (SCRF recommends [Metamask](https://metamask.io/) but any wallet is acceptable)
+* Create an account on [Request Network](https://app.request.finance/login).
 
-* funding a project
-* conference travel
-* charitable donations to an endowment in lieu
-* credits for compute resources, archive nodes, etc
-* supporting a blockchain-related organization or event of your choice
+To submit a Request invoice:
 
-Pay varies according to the complexity of the assignment and the time involved. Contributors who commit to longer-term engagements and have produced a good body of work for the forum are encouraged to seek additional and more lucrative responsibilities.
+1. Choose DAI for invoice currency.
+2. Select DAI stablecoin as the payment cryptocurrency.
+3. Add a new client using the email address invoice@scrf.io.
+4. Complete further details with the following information:
+```
+  Smart Contract Research Forum
+  1250 Broadway
+  36th Fl
+  10001 New York
+```   
```

Comment:
Its probably better to us quotes instead of code for this. 

i.e. 

`>` not back ticks